src/matrix/mod.rs

```diff
@@ -145,8 +147,11 @@ impl MatrixClient {
     }
 
     pub fn media_url(&self, url: &str) -> String {
-        let re = Regex::new("^mxc://([^/]+/[^/]+)$").unwrap();
-        if let Some(captures) = re.captures(url) {
+        lazy_static::lazy_static!{
+            static ref RE: Regex = Regex::new("^mxc://([^/]+/[^/]+)$").unwrap();
+        }
+        //let re = Regex::new("^mxc://([^/]+/[^/]+)$").unwrap();
```

Comment:
Thanks for updating this to use `lazy_static!`, but why did you leave a commented-out version of the same code in?

Reply:
Updated this in a later commit but it wasnt included in this PR. I fixed this but accidentally included it in the rustfmt commit.